components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/impl/ProvisioningClient.java

```diff
@@ -317,6 +319,110 @@ public void provisionUpdateUser() {
                                 response, SCIMConstants.identifyFormat(contentType));
                 logger.error(exception.getDescription());
             }
+        } catch (CharonException e) {
+            logger.error("Error in provisioning 'update user' operation.");
+            logger.error(e.getDescription());
+            e.printStackTrace();
+        } catch (HttpException e) {
+            logger.error("Error in provisioning 'update user' operation.");
+            logger.error(e.getMessage());
+            e.printStackTrace();
+        } catch (IOException e) {
+            logger.error("Error in provisioning 'update user' operation.");
+            logger.error(e.getMessage());
+            e.printStackTrace();
+        } catch (BadRequestException e) {
+            logger.error("Error in provisioning 'update user' operation.");
+            logger.error(e.getDescription());
+            e.printStackTrace();
+        }
+
```

Comment:
Can use nested exception here, since the error message is same. Better remove logger.error(e.getMessage()), e.printStackTrace() and logger.error(e.getDescription()) and pass the it to logging framework as follows.
logger.error("Error in provisioning 'update user' operation.", e);